src/contexts/UserContext/UserContext.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * UserContext
+ *
+ * User context provider for the application.
```

Comment:
this is a good example of a code you must write good documentation for. Explaining why this context is needed/ what purpose it solves.